src/services/VaultService/fantom.ts

```diff
@@ -1,193 +1,124 @@
-import { Yearn } from '@yfi/sdk';
-import { utils } from 'ethers';
 import memoize from 'lodash/memoize';
-import { JsonRpcProvider } from '@ethersproject/providers';
+import { RoboSdk } from '../robo-sdk';
+import { Subgraph } from '../subgraph';
+import { utils } from 'ethers';
 import { getEthersDefaultProvider } from '../../utils/ethers';
 import { mapVaultDataToVault } from '../../utils/vaultDataMapping';
-import { querySubgraphData } from '../../utils/apisRequest';
-import { mapVaultSdkToVaultApi, sortVaultsByVersion } from './mappings';
+import { sortVaultsByVersion } from './mappings';
+import getNetworkConfig from '../../utils/config';
 
 import {
     NetworkId,
     Network,
     VaultService,
     Vault,
     VaultApi,
-    StrategyApi,
     QueryParam,
     DEFAULT_QUERY_PARAM,
     StrategyMetaData,
+    StrategyWithReports,
 } from '../../types';
 
-type StrategyBasicData = {
-    [vault: string]: StrategyApi[];
-};
-
-type VaultGQLResult = {
-    id: string;
-    strategies: StrategyApi[];
-};
-
-type GQLResult = {
-    data: {
-        vaults: VaultGQLResult[];
-    };
-};
-
 export default class FantomService implements VaultService {
```

Comment:
The file has been revamped to work for Robo Vault. Data access is done via the new class `RoboSdk`.